### ISSUE TITLE
about the training loss

### ISSUE THREAD

**Issue by gelang93** [tkipf/gcn#31]
When my label is a high-dimensional vector, and the label is not sparse. I find that the training loss has not been reduced（or the reduction is particularly small）,and the end result is not good.how to solve the problem ？ Thanks

**Comment by tkipf** [comment]
Sounds like a regression problem to me. Have you tried mean-squared error (MSE) instead of cross entropy?
